### ISSUE TITLE
The `speakerphone` icon is announced to screen reader

### ISSUE THREAD

**Issue by metamoni** [joemasilotti/railsdevs.com#554]
## Describe the bug
The `speakerphone` icon in `app/components/developers/new_fields_component.html.erb` and `app/components/developers/invisible_banner_component.html.erb` is not hidden from screen readers. As a result, screen readers announce it as "image" without a description or any additional context. Because this image is purely decorative, it should be hidden from screen readers.

## To reproduce

1. Make sure your developer profile is incomplete
2. Visit the [homepage](https://railsdevs.com/)
3. Turn VoiceOver on: `command + F5`
4. Read the page contents with `control + option + Right Arrow`
5. Notice the speakerphone icon is being announced as "image"
6. Turn VoiceOver off: `command + F5`

## Expected behavior
VoiceOver should skip the speakerphone icon

## Actual behavior
The speakerphone icon is being announced as "image", without a description or any additional context

## Screenshots
https://user-images.githubusercontent.com/22390758/178144191-6a566a36-3330-429f-ba64-821f8f19d0b7.mov

## Additional context
- [Accessibility in inline_svg](https://github.com/jamesmartin/inline_svg#accessibility)
- [More on accessible SVGs](http://www.a11yonrails.com/images#accessible-svgs)

**Comment by joemasilotti** [comment]
Good idea – all yours!

**Comment by joemasilotti** [comment]
Closed by #562.
